src/chart/sankey/SankeySeries.ts

```diff
@@ -298,6 +301,12 @@ class SankeySeriesModel extends SeriesModel<SankeySeriesOption> {
             fontSize: 12
         },
 
+        edgeLabel: {
+            show: false,
+            position: 'inside',
```

Comment:
Should we limit the position to be 'inside' here? I think other position configurations will not fit here since the calculation is based on the bounding box

Reply:
@pissang Do you mean only `inside` applied?